openwisp_users/tests/test_admin.py

```diff
@@ -322,3 +322,10 @@ def test_organization_owner(self):
         self._create_org_owner()
         response = self.client.get(reverse('admin:openwisp_users_organizationowner_changelist'))
         self.assertContains(response, 'tester')
+
+    def test_organization_id(self):
```

Comment:
rename to `test_organization_uuid_field`

Reply:
Done